app/models/ontology.rb

```diff
@@ -195,6 +195,23 @@ def versioned_locid
     current_version.locid
   end
 
+  def file_deleted?(commit_oid = nil)
```

Comment:
You could document the meaning of `commit_oid` for these methods. I assume this checks if the file doesn't exist in the state of this commit (and defaults to HEAD if `commit_oid` is `nil`)?